modules/windows-containers-release-notes-limitations.adoc

```diff
@@ -29,6 +29,8 @@ Note the following limitations when working with Windows nodes managed by the WM
 
 * {productwinc} does not support adding Windows nodes to a cluster through a trunk port. The only supported networking configuration for adding Windows nodes is through an access port that carries traffic for the VLAN.
 
+* {productwinc} does not support an Windows operating system language other than English (United States). 
```

Comment:
I feel more comfortable with the definition as: {productwinc} does not support any Windows operating system language rather than English (United States).
Or in a positive manner: {productwinc} supports only Windows English (United States).

Reply:
@rrasouli I love that suggestion. Thank you.